ai_diffusion/ui/settings.py

```diff
@@ -299,7 +299,15 @@ def value(self):
     def value(self, v):
         self._checkbox.setChecked(v)
 
-
+class QMenu(QMenu):
```

Comment:
The sub-class isn't really needed here (and I don't like that it has the same name)
Please just keep the function, name it `_menu_width` or similar and call it directly